CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.11.1 (2021-04-29)
+
+- Fixed the lookup component to properly respond to reactive form control changes. [#169](https://github.com/blackbaud/skyux-lookup/pull/169)
```

Comment:
```suggestion
- Fixed the lookup component to respond to changes in reactive form controls. [#169](https://github.com/blackbaud/skyux-lookup/pull/169)
```
Might just be me, but I had trouble understanding the cluster of words at the end. I think this tweak makes it clearer.

Reply:
@blackbaud-johnly how about “ Fixed the lookup component to respond to changes in reactive form control values.”

That I calls out that the issue was the values not being responded to and not that it was that we weren’t responding to the actual controls themselves changing.

Reply:
```suggestion
- Fixed the lookup component to respond to changes in reactive form control values. [#169](https://github.com/blackbaud/skyux-lookup/pull/169)
```
I think that works great. Sorry I missed the response yesterday.